sorc/ncep_post.fd/CALVIS_GSD.f

```diff
@@ -136,6 +142,7 @@ SUBROUTINE CALVIS_GSD(CZEN,VIS)
 !                Aerosol method (2), 
 !                Smoke added to RH method for clear air  (3)
 !                   3 - option to add reducted visibility from smoke-based aerosols.
+      method_blsn = 1 ! 1 for 'on', 0 for 'off'
```

Comment:
@EricJames-NOAA I am curious if the setting for "method_blsn" can be read from the namelist "nampgb" in WRFPOST.f. You may refer to "gtg_on" or "aqf_on" and set turning on "method_blsn" as the default.

Reply:
@WenMeng-NOAA thanks for the suggestion...I will look into this option!